packages/dds/tree/src/test/simple-tree/utils.ts

```diff
@@ -110,35 +89,18 @@ export function describeHydration(
 
 /**
  * Given the schema and initial tree data, returns a hydrated tree node.
- *
+ * @remarks
  * For minimal/concise targeted unit testing of specific simple-tree content.
  *
- * TODO: determine and document if this produces "cooked" or "marinated" nodes.
+ * This this produces "marinated" nodes, meaning hydrated nodes which may not have an inner node cached yet.
  */
 export function hydrate<const TSchema extends ImplicitFieldSchema>(
 	schema: TSchema,
-	initialTree: InsertableTreeFieldFromImplicitField<TSchema>,
+	initialTree: InsertableField<TSchema> | InsertableTreeFieldFromImplicitField<TSchema>,
 ): TreeFieldFromImplicitField<TSchema> {
-	const forest = buildForest();
-
-	const branch = createTreeCheckout(testIdCompressor, mintRevisionTag, testRevisionTagCodec, {
-		forest,
-		schema: new TreeStoredSchemaRepository(toStoredSchema(schema)),
-	});
-	const manager = new MockNodeIdentifierManager();
-	const checkout = new CheckoutFlexTreeView(branch, defaultSchemaPolicy, manager);
-	const field = checkout.flexTree;
-	branch.forest.anchors.slots.set(
-		SimpleContextSlot,
-		new HydratedContext(normalizeFieldSchema(schema).allowedTypeSet, checkout.context),
-	);
-	assert(field.context.isHydrated(), "Expected LazyField");
-	const mapTree = prepareForInsertion(initialTree as InsertableContent, schema, field.context);
-
-	if (mapTree === undefined) return undefined as TreeFieldFromImplicitField<TSchema>;
-	const cursor = cursorForMapTreeField([mapTree]);
-	initializeForest(forest, cursor, testRevisionTagCodec, testIdCompressor, true);
-	return getTreeNodeForField(field) as TreeFieldFromImplicitField<TSchema>;
+	const view = getView(new TreeViewConfiguration({ schema, enableSchemaValidation: true }));
+	view.initialize(initialTree as InsertableField<TSchema>);
```

Comment:
Using the actual production initialize avoids an issue where if (in the future) we enable schema checking in the forest, the old code would have gone out of schema due to either updating the schema or the forest first.

Initialize avoids this by doing 3 phase updates if the root schema is required (schema to optional root, content update, schema to required root).

This was the motivating factor for this change as it fixes a blocking issue for running many tests with increased validation.